.github/workflows/golangci-lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: golangci-lint
```

Comment:
`go-version: 1.14.6` on L17.. maybe we should update that too? I know its just for the linter but we should stay consistent.

Reply:
Good catch. Fixed.